mlir/lib/Dialect/MIOpen/Tuning/GridwiseGemmParams.cpp

```diff
@@ -268,6 +268,12 @@ LogicalResult PopulateParamsXDL::populateDerived(
     return failure();
   }
 
+  res = isKpackValid(&params, gemmADerivedParam, gemmBDerivedParam);
+  if (failed(res)) {
+    LLVM_DEBUG(llvm::dbgs() << "Incoherent kpack tuning parameter.\n");
+    return failure();
+  }
```

Comment:
Question: Should I check this in padding kernel?

Reply:
@whchung clarified that padding kernel will not use kpack feature.

Therefore, it is safe to skip this check in padding kernel.